src/Accelerators.jl

```diff
@@ -0,0 +1,47 @@
+# should be included in EnsembleKalmanProcess.jl
+
+export DefaultAccelerator, NesterovAccelerator
+
+
+# Since nothing is a possible input to EnsembleKalmanProcess, we do not need a default accelerator (default is nothing)
+"""
+$(TYPEDEF)
+
+Default accelerator provides no acceleration.
+
+$(TYPEDFIELDS)
+"""
+struct DefaultAccelerator <: Accelerator end # can use types/subtypes to ensure v, ens_prev consistent, and to multiple dispatch if multiple cases.
+
+function DefaultAccelerator()
```

Comment:
You should remove this constructor - as (1) The `DefaultAccelerator()` constructor is automatically created when defining the struct above, and (2) we do not want this to return the Nesterov Accelerator.

Reply:
So when we define the struct, there is a constructor automatically made for an object literally called "DefaultAccelerator"?